auto_py_to_exe/web/js/i18n.js

```diff
@@ -1226,7 +1236,7 @@ const translationMap = {
                 zh_tw: '轉換中······',
                 ua: 'Конвертація...',
                 ru: 'Преобразование...',
-                de: 'Konvertierung...',
+                de: 'Konvertierung ...',

```

Comment:
@ackhh is this extra space needed?

Reply:
as I now look at it ... no, no extra space - sorry